src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,18 +25,38 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        Country austria = new Country("AUS");
+        CountryDao countryDao = new CountryDaoImpl(sessionFactory);
```

Comment:
It's important to ensure that the application does not commit real database credentials. While this file does not contain credentials, make sure that `HibernateUtil` or any other configuration files are using placeholder credentials as per the checklist.